src/bionics.cpp

```diff
@@ -546,11 +547,28 @@ bool player::activate_bionic( int b, bool eff_only )
         if( !has_cable ) {
             add_msg_if_player( m_info,
                                _( "You need a jumper cable connected to a vehicle to drain power from it." ) );
+        } else {
+            const optional_vpart_position vp_pos = g->m.veh_at( pos() );
+            if( vp_pos ) {
+                vehicle *const veh = &vp_pos->vehicle();
```

Comment:
Any reasons you're using a pointer here and not a reference (what `vehicle()` returns)?

Reply:
I think that's how the code I swiped it from did it.